coprocessor/fhevm-engine/coprocessor/src/server.rs

```diff
@@ -895,3 +901,27 @@ impl CoprocessorService {
         Ok(tonic::Response::new(result))
     }
 }
+
+async fn sort_computation_into_bucket(
+    computations: &[&crate::server::coprocessor::AsyncComputation],
+) -> Vec<Vec<u8>> {
```

Comment:
Maybe you declare `type Handle = Vec<u8>; type BucketId = Vec<u8>;` to make it easier to read.